src/coreclr/jit/compiler.h

```diff
@@ -11046,6 +11046,34 @@ class GenTreeVisitor
                 break;
 #endif // defined(FEATURE_SIMD) || defined(FEATURE_HW_INTRINSICS)
 
+            case GT_SELECT:
+            case GT_CEQ:
+            case GT_CNE:
+            case GT_CLT:
+            case GT_CLE:
+            case GT_CGE:
+            case GT_CGT:
```

Comment:
That did occur to me when copy/pasting the GenTreeCmpXchg code. I suspect that's going to cause quite a large diff through the code (not sure if there's any other nodes that could become a GenTreeTriOp too).

Would it be ok to do that as a follow on pull request? Should be fairly easy to do by itself, just a lot of diff. And I'd rather not squash it into this PR.

Reply:
> Would it be ok to do that as a follow on pull request?

I would be fine with that. The code diff will not be small, but I don't think that's a big problem (since it's going to also be "mechanical").

> not sure if there's any other nodes that could become a GenTreeTriOp too

Yes, there's also `GenTreeArrOffs`.